package-examples/helloworld/README.md

```diff
@@ -0,0 +1,53 @@
+# helloworld
+
+Sample helloworld package.  Showcases what a bare bones package is which doesn't do anything beyond declaring the current directory as a `kpt` package. 
+
+## Steps
+
+This is a simple workflow on a package that requires no configuration or
+customization.
+
+1. [Fetch the package](#fetch-the-package)
+2. [View the package contents](#view-the-package-contents)
+3. [Apply the package](#apply-the-package)
+
+### Fetch the package
+
+Get the example package on to local using `kpt pkg get`
+
+  $ kpt pkg get https://github.com/GoogleContainerTools/kpt.git/package-examples/helloworld
+
+    fetching package /package-examples/helloworld from https://github.com/GoogleContainerTools/kpt to helloworld
+
+### View the package contents
+
+List the package contents in a tree structure.
+
+  $ kpt pkg tree helloworld/
+
+    helloworld
```

Comment:
This should be PKG: helloworld. Pull the latest upstream/next and try.

Reply:
updated